requirements.yml

```diff
@@ -15,3 +15,6 @@
 
 - src: geerlingguy.ntp
   version: "1.6.2"
+
+- src: andrewrothstein.anaconda
+  version: "v4.0"
```

Comment:
This isn't a valid version, `v4.0.8` works

Reply:
Also - is there a reason to use Anaconda, or will miniconda suffice? Anaconda is a lot larger.

Reply:
No strong reason for using anaconda vs miniconda afaik. I’ll see if I can swap this out and fix role dependencies.

I’ll also test the CUDA install issue. I suspect this is a CentOS vs Ubuntu issue.

Have meetings this morning but will likely push an updated version early this afternoon.

(I wish Github let me mark “needs changes” on my own PR, it’s a good way to flag that there are changes pending!)